dev/com.ibm.ws.springboot.fat20_fat/fat/src/com/ibm/ws/springboot/support/fat/utility/SpringBootUtilityThinTest.java

```diff
@@ -370,6 +371,18 @@ public void testDefaultHostWithAppPortRunLibertyUberJarWithSSL() throws Exceptio
                 line = readTimeout(reader);
             }
         }
+
+        //Debug the failure within the JAR:
```

Comment:
Should write a `Log.info` here indicating the path `libertyUberJar.getAbsolutePath()` and that the following is the content of the uber JAR.  Something like:
```java
Log.info(getClass(), "testRunLibertyUberJarWithSSL", "The following is the content of the uber Jar: " + libertyUberJar.getAbsolutePath());
```